src/child/components/favourite/Favourite.js

```diff
@@ -180,9 +180,9 @@ class Favourite extends Component {
                             <div className="details">
                                 {price && <div className="price">{price}</div>}
                                 {delta && <div className="delta">{delta}</div>}
-                                {percentage && <div className="percentage">
+                                {percentage !== null && <div className="percentage">
```

Comment:
Personally, I would not rely on null or any specific value, as it needs to be synchronized with the test you make when defining `percentage`